src/solvers/dgmulti/types.jl

```diff
@@ -225,6 +225,24 @@ function DGMultiMesh(dg::DGMulti{NDIMS}, cells_per_dimension, mapping;
   return DGMultiMesh(dg, GeometricTermsType(Curved(), dg), md_curved, boundary_faces)
 end
 
+"""
+    DGMultiMesh(dg::DGMulti, filename::String)
+
+- `dg::DGMulti` contains information associated with to the reference element (e.g., quadrature,
```

Comment:
small typo: decide whether you take "associated with" or "associated to" :smile:

Reply:
Oops, thanks for the catch!